nano/node/election.cpp

```diff
@@ -636,3 +652,30 @@ std::vector<nano::vote_with_weight_info> nano::election::votes_with_weight () co
 	std::transform (sorted_votes.begin (), sorted_votes.end (), std::back_inserter (result), [] (auto const & entry) { return entry.second; });
 	return result;
 }
+
+nano::election_behavior nano::election::get_behavior () const
+{
+	return behavior;
+}
+
+nano::stat::detail nano::to_stat_detail (nano::election_behavior behavior)
```

Comment:
The conversions when handling `behaviour` would be a good candidate for a separate PR. I'm thinking the if -> switch conversions and utility functions that were added.